backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/production/productionCandidate.feature

```diff
@@ -146,5 +109,94 @@ Feature: Production dispo scenarios
     Then update PP_Order_Candidate's qty entered to less than processed expecting exception
       | PP_Order_Candidate_ID.Identifier | QtyEntered |
       | oc_1                             | 2          |
-    
-    
\ No newline at end of file
+
+  @Id:S0129.2_190
+  @from:cucumber
+  Scenario: Reactivate and reduce QTY for order with negative ATP and PreparationDate in the past. Ensure correct qty is used on new PP_Order_Candidate.
+    When metasfresh has date and time 2025-02-25T07:00:00+01:00[Europe/Berlin]
+    And metasfresh contains M_PricingSystems
+      | Identifier |
+      | ps_1       |
+    And metasfresh contains M_PriceLists
+      | Identifier | M_PricingSystem_ID | C_Country_ID | C_Currency_ID | SOTrx |
+      | pl_1       | ps_1               | DE           | EUR           | true  |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID |
+      | plv_1      | pl_1           |
+    And metasfresh contains M_ProductPrices
+      | M_PriceList_Version_ID | M_Product_ID | PriceStd | C_UOM_ID | C_TaxCategory_ID |
+      | plv_1                  | p_1          | 10.0     | PCE      | Normal           |
+    And metasfresh contains C_BPartners:
+      | Identifier | IsVendor | IsCustomer | M_PricingSystem_ID |
+      | customer   | N        | Y          | ps_1               |
+    And metasfresh contains C_BPartner_Locations:
+      | Identifier | GLN          | C_BPartner_ID |
+      | location_1 | bPLocation_1 | customer      |
```

Comment:
i suggest not using GLN if not needed because
* cucumber is looking up for that GLN => so basically test depends on other tests => fragile
* instead set the IsBillTo, IsBillToDefault, IsShipTo, IsShipDefault

Reply:
done